function-support.html

```diff
@@ -321,6 +319,43 @@ <h2 id="accents">Accents</h2>
 			<td></td>
 		</tr>
 	</table>
+	
+	<h4 id="accents-inside-\text…}">Accents inside \text{…}</h4>

```

Comment:
This ID could maybe use a tweak -- either an `{` or maybe a simplification to avoid fancy Unicode characters...